django/template/base.py

```diff
@@ -201,6 +201,24 @@ def render(self, context):
         finally:
             context.render_context.pop()
 
+    def _stream(self, context):
+        return self.nodelist.stream(context)
+
+    def stream(self, context):
+        "Display stage -- can be called many times"
```

Comment:
do you know the meaning of this comment? I see it's copied from elsewhere, but its meaning isn't clear to me.